mlrun/runtimes/local.py

```diff
@@ -487,6 +488,11 @@ def exec_from_params(handler, runobj: RunObject, context: MLClientCtx, cwd=None)
             else:
                 val = handler(**kwargs)
             context.set_state("completed", commit=False)
+        except mlrun.errors.MLRunTaskCancelledError as exc:
+            logger.warning("Run was aborted", err=err_to_str(exc))
+            # Run was aborted, the state run state is updated by the abort job, no need to commit again
+            context.set_state(mlrun.runtimes.constants.RunStates.aborted, commit=False)
+            commit = False
```

Comment:
looking down, logger returns to old level - dont you want to do it as well? perhaps we need to conform both behavior

Reply:
the set logger level is duplicated with one at the end of the function. I have merged them with try, finally clause